src/OrbitGl/TextRenderer.cpp

```diff
@@ -254,25 +254,24 @@ void TextRenderer::AddTextInternal(const char* text, ftgl::vec2* pen,
       float kerning = (i == 0) ? 0.0f : texture_glyph_get_kerning(glyph, text + i - 1);
       pen->x += kerning;
 
-      float x0 = floorf(pen->x + glyph->offset_x);
-      float y0 = floorf(pen->y - glyph->offset_y);
-      float x1 = floorf(x0 + glyph->width);
-      float y1 = floorf(y0 + glyph->height);
+      Vec3 pos0 = translations_.TransformAndFloorVertex(
+          Vec3(pen->x + glyph->offset_x, pen->y - glyph->offset_y, z));
+      Vec3 pos1 = pos0 + Vec3(glyph->width, glyph->height, 0);
```

Comment:
This is slightly changing the behaviour because the floor is now happening before adding glyph->width or height. I assume that's not a problem? Or does that maybe need an additional floor?

Reply:
`width` and `height` are both `size_t` so this won't make a difference.